include/picongpu/plugins/particleCalorimeter/ParticleCalorimeter.hpp

```diff
@@ -643,6 +663,8 @@ namespace picongpu
                 particles->getDeviceParticlesBox(),
                 (MyCalorimeterFunctor) * this->calorimeterFunctor,
                 mapper,
```

Comment:
Discussed offline. The mapper should not be `GUARD`, but some form of runtime mapper for the boundary. I will wait for other open PRs to be merged to `dev`, then rebase, and make an appropriate mapper in this branch,

Reply:
Fixed now.